### DISCUSSION TITLE
Student Pays Option

### DISCUSSION THREAD

**Discussion by mfbutner** [PrairieLearn/PrairieLearn#6185] [General]
Would it be possible to add a "Student Pays" option for hosted Prairie Learn? I know that this isn't the ideal option from either the student perspective or yours but I it opens up areas for adoption that might otherwise be closed. In my own personal situation, funds are tight within our department and so I don't think my department would grant a funding request for Prairie Learn in my classes. However I see so much power and potential in Prairie Learn and so it would be a shame to have to forego using it. 

A similar situation happened when I switched to Zybooks years ago but since they had a student pays I could start using immediately. This gave it a small foothold in our department and it organically spread from there so now it is used throughout our basic programming series (Intro, OO, Data Structures and Algorithms) by at least 4 different instructors. Similar things happened with GradeScope and now we have University License. 

TL;DR Having a student pays option makes it easy for earlier adopters to start using a service and then growth happens organically from there.

If this isn't the right place for this discussion, please let me know where I should move it to.
